docs/output.md

```diff
@@ -435,6 +436,29 @@ Krona charts will be generated by the pipeline for supported tools (Kraken2, Cen
 
 The resulting HTML files can be loaded into your web browser for exploration. Each file will have a dropdown to allow you to switch between each sample aligned against the given database of the tool.
 
+### TAXPASTA
+
+[TAXPASTA](https://github.com/taxprofiler/taxpasta) is a python package added as a module in nf-core/taxprofiler that can standardise and merge two or more taxonomic profiles across samples into one single table..
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `taxpasta`
+  - `<db_name>.<sample_id>*.{tsv,csv,arrow,parquet,biom}`: A list of taxonomic profiler output files. The standard format is the `tsv`. The first column describes the taxonomy ID and the rest of the columns describe the read counts for each sample.
+  </details>
+
+The following report files are used for the taxpasta step:
+
+- Bracken: `<sample>_<db_name>.tsv`
```

Comment:
Taxpasta displays the `new_est_reads` column? @sofstam @Midnighter can you confirm?

We need make it clear _which_ value (count? percent? abundance value) is being displayed for each tool (please check my comments below for all profilers).

Reply:
Should probably be documented in taxpsata too tbh...

Reply:
It's the `new_est_reads`, yes. And I agree, that should be documented on taxpasta.